src/cpp/session/modules/SessionRCompletions.R

```diff
@@ -1767,6 +1768,12 @@ assign(x = ".rs.acCompletionTypes",
    # parameterized R Markdown documents
    if (.rs.injectKnitrParamsObject(documentId))
       on.exit(.rs.removeKnitrParamsObject(), add = TRUE)
+
+   # If custom completions have been set through
+   # 'rc.options("custom.completions")',
+   # then use the internal R completions instead.
+   if (.rs.isCustomCompletionsEnabled())
+      return(.rs.getCustomRCompletions(line))
```

Comment:
Should we fall back on our on completion system if the custom completer generates an error?